src/index.css

```
@@ -176,6 +177,35 @@
+
+  .dialog-title {
+    @apply font-display text-lg font-bold tracking-wider text-base-content;
+  }
+
+  .section-subheading {
+    @apply font-display text-base font-bold tracking-wider;
+  }
+
+  .brand-heading {
+    @apply font-sans font-extrabold tracking-widest uppercase text-accent;
+    text-shadow: 0 0 20px rgba(254, 234, 0, 0.3);
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)